presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -864,18 +864,13 @@ private static boolean isAsciiPrintable(int codePoint)
         return true;
     }
 
-    private static String formatGroupingSet(Set<Expression> groupingSet, Optional<List<Expression>> parameters, int indent, SqlFormatterType sqlFormatterType)
+    private static String formatGroupingSet(List<Expression> groupingSet, Optional<List<Expression>> parameters, int indent, SqlFormatterType sqlFormatterType)
```

Comment:
don't merge upstream/master. Either rebase or don't update at all

Reply:
https://github.com/prestodb/presto/pull/11329#discussion_r212767435

Reply:
should I create a new PR?

Reply:
no need for new PR. You can rebase your `queryAbbreviation` branch on current master.